handlers/handlers.go

```diff
@@ -40,68 +42,98 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 	w.WriteHeader(status)
 }
 
-func forwardFlorenceTokenIfRequired(req *http.Request) *http.Request {
-	if len(req.Header.Get(common.FlorenceHeaderKey)) > 0 {
-		ctx := common.SetFlorenceIdentity(req.Context(), req.Header.Get(common.FlorenceHeaderKey))
-		return req.WithContext(ctx)
-	}
-	return req
+func setStatusCode404(req *http.Request, w http.ResponseWriter, err error) {
+	status := http.StatusNotFound
+	log.ErrorCtx(req.Context(), err, log.Data{"setting-response-status": status})
+	w.WriteHeader(status)
 }
 
-//GeographyRender ...
-func GeographyRender(rend RenderClient) http.HandlerFunc {
+//GeographyHomepageRender ...
+func GeographyHomepageRender(rend RenderClient) http.HandlerFunc {
```

Comment:
If you pass the config objection into this function an argument and then call create a url var in here

```
func GeographyHomepageRender(rend RenderClient, cfg config.Config) http.HandlerFunc {
    ...
    url := cfg.CodeListsAPIURL
    ...
}
```